pkg/ebpf/bytecode/bindata/bindataOffsetguessO.go

```diff
@@ -28,7 +28,7 @@ func bindataOffsetguessO() (*asset, error) {
 		name:        "offset-guess.o",
 		size:        12360,
 		md5checksum: "",
-		mode:        os.FileMode(420),
+		mode:        os.FileMode(436),
```

Comment:
Tiny nit: I don't know if these files are generated but it would be helpful to have a comment about what all these mode values are in octal right next to their use for easier visual parsing:
```suggestion
		mode:        os.FileMode(436),  // Octal: 0664
```

Reply:
They are generated, so unfortunately that would get overwritten